kv/notification_rule.go

```diff
@@ -79,6 +79,9 @@ func (s *Service) createNotificationRule(ctx context.Context, tx Tx, nr influxdb
 	nr.SetOwnerID(userID)
 	nr.SetCreatedAt(now)
 	nr.SetUpdatedAt(now)
+	if err := nr.Valid(); err != nil {
+		return err
+	}
```

Comment:
We already did the validation at line `if err := s.putNotificationRule(ctx, tx, nr)`, i don't think this is needed